lib/components/home/artist_rails/artist_card.js

```diff
@@ -33,6 +33,7 @@ class ArtistCard extends React.Component {
     ARTemporaryAPIModule.setFollowStatus(true, this.props.artist._id, (error, following) => {
       if (error) {
         __DEV__ ? console.error(error) : console.log(error)
+        this.setState({ processingChange: false })
```

Comment:
So you mean to move the last line in this method to be before the networking request, right, not this line that’s shown in the review? I think that makes sense, will do.